### DISCUSSION TITLE
Changing the default weight for Intensity=Bold

### DISCUSSION THREAD

**Discussion by hexchain** [wezterm/wezterm#2790] [Q&A] [answered]
It seems that the default font weight when Intensity=Bold is DemiBold. With JetBrains Mono, there doesn't really seem to be a difference between this and Normal:

![image](https://user-images.githubusercontent.com/529520/203790218-c7848a04-6d5c-4200-965a-69593ae3d2d7.png)

Is it possible to change this without involving `font_rules`?

**Comment by wez** [comment, marked as answer]
It looks more bold to me. Regardless, you can use `font_rules` to override this.

**Reply by hexchain** [reply]
Thanks for the reply! Indeed it looks a bit bolder than the normal ones, but very subtly.

I'm a bit hesitant about using `font_rules`, mainly because either I'll have to repeat the `wezterm.font_with_fallback {...}` call, or define a function to produce the same result. If this is the only preferred way, I'm also fine with it.
